sdk/search/Azure.Search.Documents/src/Generated/Models/AnalyzeResult.Serialization.cs

```diff
@@ -7,7 +7,6 @@
 
 using System.Collections.Generic;
 using System.Text.Json;
-using Azure.Core;
```

Comment:
If not for noting another change w.r.t. nullables, can we not do changes that would just do something meaningless like this? An extra `using` is rarely a problem, and our source is already littered with them anyway. It's disruptive.

Reply:
This is a side effect of the ongoing consolidation work. Old code has been "guessing" in some cases that we may need `Azure.Core` reference and has been adding it even when it is not used. New code tracks all types directly and adds namespaces only when they are needed.

Reply:
That's fine. Just for a bit it seemed like that was the only change, which seems overkill for a deployment.